### PR TITLE
Don't block the explorer during installations

### PR DESCRIPTION
**Pull request by Megamouse**
This should fix the following issue:
When using drag&drop to install stuff the windows explorer instance was useless until the installation was complete.

### CODE REVIEW

File: rpcs3/rpcs3qt/main_window.cpp [hunk 1]
```diff
@@ -397,6 +397,18 @@ void main_window::InstallPackages(QStringList file_paths, bool show_confirm)
 		}
 	}
 
+	if (!file_paths.isEmpty())
+	{
+		// Handle the actual installations with a timeout. Otherwise the source explorer instance is not usable during the following file processing.
+		QTimer::singleShot(100, [this, file_paths]()
```
Comment (MSuih) [inline rpcs3/rpcs3qt/main_window.cpp]: Have you tried setting the delay to 0 ms? It should make it fire as soon as the current function returns. I don't see any reason to specifically wait 100 milliseconds.

Comment (Megamouse) [reply rpcs3/rpcs3qt/main_window.cpp]: No. Die you?

Comment (MSuih) [reply rpcs3/rpcs3qt/main_window.cpp]: Haven't had time to test it, but I thought i'd give a suggestion on how to improve this. No need to get offended.

Comment (Megamouse) [reply rpcs3/rpcs3qt/main_window.cpp]: what do you mean

Comment (Megamouse) [reply rpcs3/rpcs3qt/main_window.cpp]: I just wanted to know if it works before I change it. But I tried later and it did.

Comment (MSuih) [reply rpcs3/rpcs3qt/main_window.cpp]: Oh sorry, thought you were being snarky because I left it as a comment without having tested it myself.

**Pull request merged**